packages/server/lib/plugins/child/require_async_child.js

```diff
@@ -11,6 +22,15 @@ const util = require('../util')
 const ipc = util.wrapIpc(process)
 const run = require('./run_require_async_child')
 
-const { file, projectRoot } = require('minimist')(process.argv.slice(2))
+exporter.attachIPC(ipc)
+
+ipc.on('main:process:will:disconnect', async () => {
+  if (span) {
+    span.end()
+  }
```

Comment:
```suggestion
span?.end()
```

Reply:
i wasn't sure if optional chaining exists in our minimum node version for the child process, but it looks like our minimum is node 14 and optional chaining exists there so this should be ok.

Reply:
unless it breaks a bunch of tests, then i'll roll it back.

Reply:
i rolled this back, it did fail tests.